run.go

```diff
@@ -86,6 +86,8 @@ type RunOptions struct {
 	Runtime string
 	// Args adds global arguments for the runtime.
 	Args []string
+	// NoHostname use the images /etc/hostname file
```

Comment:
"image's", but the wording is still kind of awkward, since images aren't required to provide a /etc/hostname, so a given image might not include one.